### PR TITLE
update names

### PR DESCRIPTION
**Pull request by bentan2013**

### CODE REVIEW

File: src/ArcGISEarth.AutoAPI.Utils/AutomationAPIHelper.cs [hunk 1]
```diff
@@ -23,19 +22,20 @@ namespace ArcGISEarth.AutoAPI.Utils
 {
     public class AutomationAPIHelper
     {
-        private const string TARGET_OPERATIONALLAYERS = "operationalLayers";
+        public const string CameraControllerName = "camera";
+        public const string FlightControllerName = "flight";
+        public const string LayerControllerName = "layer";
+        public const string LayersControllerName = "layers";
+        public const string WorkspaceControllerName = "workspace";
+        public const string SnapshotControllerName = "snapshot";
```
Comment (gis2all) [inline src/ArcGISEarth.AutoAPI.Utils/AutomationAPIHelper.cs:30]: `public`  ->  `private`

Comment (geoshow) [reply src/ArcGISEarth.AutoAPI.Utils/AutomationAPIHelper.cs:30]: use `public` so other class can use this. the same logic as the `APIBaseUrl` @gis2all

Comment (bentan2013) [reply src/ArcGISEarth.AutoAPI.Utils/AutomationAPIHelper.cs:30]: use public so other class can use this. the same logic as the public `get` for  `APIBaseUrl` @gis2all

Comment (bentan2013) [reply src/ArcGISEarth.AutoAPI.Utils/AutomationAPIHelper.cs]: already change `public` to `priivate` and merged this branch

**Pull request merged**